PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -195,10 +195,10 @@ def nanoAOD_recalibrateMETs(process,isData):
         table.variables.muonSubtrFactor = Var("1-userFloat('muonSubtrRawPt')/(pt()*jecFactor('Uncorrected'))",float,doc="1-(muon-subtracted raw pt)/(raw pt)",precision=6)
     process.metTables += process.corrT1METJetTable
 #    makePuppiesFromMiniAOD(process,True) # call this before in the global customizer otherwise it would reset photon IDs in VID
```

Comment:
can we clean up this call ?

Reply:
I have now modified this part to explicitly set as True the useExistingWeights as per the previous comment thread